### DISCUSSION TITLE
comment resoudre cette erreur?

### DISCUSSION THREAD

**Discussion by mimiossart** [project-error/npwd#1173] [Q&A]
[           resources] Stopping resource npwd
[      resources:npwd] Warning: could not find client_script resources/dist/client/*.client.js (defined in fxmanifest.lua:6)
[      resources:npwd] Warning: could not find server_script resources/dist/server/*.server.js (defined in fxmanifest.lua:11)
[      resources:npwd] Warning: could not find file resources/html/index.html (defined in fxmanifest.lua:17)

**Comment by stevenbreak** [comment]
Salut tu na pas prit la release ... tu as prit le dossier a l'etat pur ^^

prend sa 
https://github.com/project-error/npwd/releases/tag/3.15.0

**Comment by mimiossart** [comment]
pourrait tu m'aider a l'installer


Le lun. 26 août 2024 à 18:13, stevenbreak ***@***.***> a
écrit :

**Reply by stevenbreak** [reply]
salut deja telecharge avec le lien que je t'ai donner apres c'est assez simple faut lire la doc si j'ai reussi tous le monde le peut ^^
